### ISSUE TITLE
"zooming font size"

### ISSUE THREAD

**Issue by jirilebl** [tomscii/zutty#114]
Getting older, my eyesight is not what it used to be.  Often, especially when tired in the evening, I need bigger font sizes, but I don't want to have big font sizes all the time.  (Also when connected to large monitor, I don't need large font).  It would be great to be able to dynamically change font size instead of having to do an entire song and dance to launch a new instance and then move the session with tmux, which is a pain in the butt.  E.g., st allows this.

**Comment by tomscii** [comment]
You have my sympathy. That said, there is a FAQ for Zutty and it is very clear on this (I hope).

What you can possibly do is (especially if you would be satisfied with a few, like normal-bigger-huge) to have a couple configurations in parallel (see the docs) and invoke them with some keybinding or other desktop mechanism, which would give you a new terminal with the configured font. So e.g. binding-1 would open zutty-normal, binding-2 would open zutty-bigger, etc.

**Comment by jirilebl** [comment]
I know I can do THAT, but that is no good.  I really need to zoom that SAME terminal, I already have the work started in that terminal, I want to read THAT, not some other terminal where I wasn't doing anything yet.  Yes I can start a new terminal with larger font and move the tmux session to a new terminal just to see "what is that word over there".  That is just too much.  Or to move a bunch of sessions just because I connected a monitor.  Nah ...

Oh well, no worries, I am back to st.

(By the way, not sure which FAQ you mean...)

**Comment by tomscii** [comment]
1. I have specifically referred to [this FAQ item](https://github.com/tomscii/zutty/wiki/FAQ#q-why-would-i-not-want-to-use-zutty). (Hint: browse github or employ the most trivial web search to find the FAQ; whining as a reaction to difficulties is generally less accepted in grown-ups.)
2. You are using a piece of software that has clearly stated limitations.
3. You have opened a ticket against one of the clearly stated "out of scope" limitations, when you should have understood at the outset that the program is not aiming to cater for your specific needs. Because, you know, not everyone in the world is out to cater for the very needs of your entitled little self. Deal with it.
4. Despite of all the above, you have STILL gotten a reply in good faith with a possible workaround.
5. You don't like it, so you come back here and throw a tantrum. (Hint: I don't care if you use st or whatnot, just as I am utterly indifferent towards all other aspects of your life.)

Your self-entitled attitude stinks. Your approach is hostile. Go away.

**Comment by jirilebl** [comment]
Oh wow, you must be fun at parties.
